desktop/cmp/treemap/SplitTreeMapModel.ts

```diff
@@ -0,0 +1,197 @@
+    get error()             {return this.primaryMapModel.error}
+    get emptyText()         {return this.primaryMapModel.emptyText}
+    get highchartsConfig()  {return this.primaryMapModel.highchartsConfig}
+    get labelField()        {return this.primaryMapModel.labelField}
+    get heatField()         {return this.primaryMapModel.heatField}
+    get maxDepth()          {return this.primaryMapModel.maxDepth}
+    get maxHeat()           {return this.primaryMapModel.maxHeat}
+    get algorithm()         {return this.primaryMapModel.algorithm}
+    get colorMode()         {return this.primaryMapModel.colorMode}
+    get theme()             {return this.primaryMapModel.theme}
+
+    @action
+    setOrientation(orientation: SplitTreeMapOrientation) {
+        this.orientation = orientation;
+    }
+
+    @action
+    setHighchartsConfig(highchartsConfig: any) {
+        this.primaryMapModel.setHighchartsConfig(highchartsConfig);
+        this.secondaryMapModel.setHighchartsConfig(highchartsConfig);
+    }
+
+    @action
+    setLabelField(labelField: string) {
+        this.primaryMapModel.setLabelField(labelField);
+        this.secondaryMapModel.setLabelField(labelField);
+    }
+
+    @action
+    setHeatField(heatField: string) {
+        this.primaryMapModel.setHeatField(heatField);
+        this.secondaryMapModel.setHeatField(heatField);
+    }
+
+    @action
+    setMaxDepth(maxDepth: number) {
+        this.primaryMapModel.setMaxDepth(maxDepth);
+        this.secondaryMapModel.setMaxDepth(maxDepth);
+    }
+
+    @action
+    setMaxHeat(maxHeat: number) {
+        this.primaryMapModel.setMaxHeat(maxHeat);
+        this.secondaryMapModel.setMaxHeat(maxHeat);
+    }
+
+    @action
+    setAlgorithm(algorithm: TreeMapAlgorithm) {
+        this.primaryMapModel.setAlgorithm(algorithm);
+        this.secondaryMapModel.setAlgorithm(algorithm);
+    }
+
+    @action
+    setColorMode(colorMode: TreeMapColorMode) {
+        this.primaryMapModel.setColorMode(colorMode);
+        this.secondaryMapModel.setColorMode(colorMode);
+    }
+
+    @action
+    setTheme(theme: Theme) {
+        this.primaryMapModel.setTheme(theme);
+        this.secondaryMapModel.setTheme(theme);
+    }
+
+    //-------------------------
+    // Implementation
+    //-------------------------
+    defaultMapFilter = (record) => {
+        const data = record instanceof StoreRecord ? record.data : record;
+        return data[this.primaryMapModel.valueField] >= 0;
+    };
+}
+
+/**
+ * Return true if record belongs to / should appear within the primary map, falsy to
+ * have it allocated to the secondary map.
+ */
+type SplitTreeMapFilterFn = (record: RawData|StoreRecord) => boolean;
+
+/**
+ * Function to generate the region title to display.
+ * `isPrimary` is true if the region is the primary (top/left) map in the pair.
+ */
+type SplitTreeMapTitleFn = (treeMapModel: TreeMapModel, isPrimary:boolean) => string|ReactNode;
```

Comment:
missing space after isPrimary/. ReactNode should include string, no need to union.

Reply:
also possible to just define these inline in the two places they appear.